modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  trigger:
+    options:
+      disabled: false
+      configMaps:
+        config-defaults-triggers: 
+          data:
+            default-run-as-non-root: "true"
+            default-run-as-user: "65532"
+            default-run-as-group: "65532"
+            default-fs-group: "65532"
+----
++
+Optionally, you can set the values of the `default-run-as-user` and `default-run-as-group` parameters to configure the numeric user ID and group ID for running the event listeners in containers; {pipelines-shortname} sets these values in the pod security context and container security context for event listeners. If you use empty values, the default user ID and group ID of `65532` are used.
++
+You can also set the `default-fs-group` parameter to define the `fsGroup` value for the pod security context, which is the group ID that the container processes use for the file system. If you use an empty value, the default group ID of `65532` is used.
+
+* With this update, in pipelines triggers, the `EventListener` pod template now includes `securityContext` settings. Under these settings, you can configure `seccompProfile`, `runAsUser`, `runAsGroup`, and `fsGroup` parameters when the `el-security-context` flag is set to `true`. 
+
+[id="web-console-new-features-1-16_{context}"]
+=== Web console
+
+* Before this release, when using the web console, you could not see the timestamp for the logs that {pipelines-shortname} created. With this update, the web console includes timestamps for all logs for {pipelines-shortname}.
+
+* With this update, the pipeline run and task run *list* pages in the web console now have a filter for the datasource, that is `k8s` and `TektonResults API`.
+
+* Before this update, when using the web console in the *Developer* perspective, you could not specify the timeout for pipeline runs. With this update, you can now set timeout while starting the pipeline run in the *Developer* perspective of the web console. 
+
+* Before this update, the *Overview* pipeline dashboard only appeared when {tekton-results} was enabled. All the statistics came from only the results API. With this update, the *Overview* pipeline dashboard is available regardless of whether {tekton-results} is enabled or not. When {tekton-results} are disabled, you can use the dashboard to see the statistics for objects in the cluster.
+
+* With this update, the sample pipelines displayed in the web console use the `v1` version of the {pipelines-shortname} API.
+
+[id="cli-new-features-1-16_{context}"]
+=== CLI
+
+* With this update, the `tkn customrun delete <custom_run_names>` command is added to delete one or more custom runs.
+
+* With this update, when you run a `list` command with the `-o` YAML flag, the listed resources are now separated with `---` separators to enhance readability of the output.
```

Comment:
presumably `tkn list` ?

Reply:
@mramendi I wrote it as just `list`, because there are (when looking at our tkn reference docs) `tkn pipeline list`, `tkn task list`, etc. So I thought having it as `tkn list` would not be accurate.  Maybe `tkn <resource> list`? Which do you think would be better?

Reply:
Great! Done